docs/zkapps/tutorials/10-account-updates.mdx

```diff
@@ -214,57 +235,56 @@ You declare functions for initializing the account and the `add()` method that i
 
 Now it's time to learn about the [main.ts](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/10-account-updates/src/main.ts) file that creates transactions with the earlier smart contracts and the account update visualizations it creates.
 
-First, import the transaction visualizer:
+1. Import the transaction visualizer:
 
-```typescript
-...
-import { showTxn, saveTxn, printTxn } from 'mina-transaction-visualizer';
-...
-```
-
-This provides three functions:
+  ```typescript
+  ...
+  import { showTxn, saveTxn, printTxn } from 'mina-transaction-visualizer';
+  ...
+  ```
 
-```typescript
-// creates a png file of a transaction, and opens it in a local image viewer
-async showTxn(txn: Mina.Transaction, name: string, legend: Legend)
+  This provides three functions:
 
-// creates a png file of a transaction, and saves it to a path
-saveTxn(txn: Mina.Transaction, name: string, legend: Legend, path: string)
+  ```typescript
+  // creates a png file of a transaction, and opens it in a local image viewer
+  async showTxn(txn: Mina.Transaction, name: string, legend: Legend)
 
-// prints a nicely formatted view of a transaction
-printTxn(txn: Mina.Transaction, name: string, legend: Legend)
+  // creates a png file of a transaction, and saves it to a path
+  saveTxn(txn: Mina.Transaction, name: string, legend: Legend, path: string)
 
-// with legend type, to replace public keys with human readable strings:
-type Legend = { [pk: string]: string };
+  // prints a nicely formatted view of a transaction
+  printTxn(txn: Mina.Transaction, name: string, legend: Legend)
 
-```
+  // with legend type, to replace public keys with human readable strings:
+  type Legend = { [pk: string]: string };
+  ```
 
-Now, define the legend as follows:
+1. Next, define the legend as follows:
 
-```typescript
-const legend = {
-  [proofsOnlyAddr.toBase58()]: 'proofsOnlyZkApp',
-  [secondaryAddr.toBase58()]: 'secondaryZkApp',
-  [deployerAccount.toPublicKey().toBase58()]: 'deployer',
-};
-```
+  ```typescript
+  const legend = {
+    [proofsOnlyAddr.toBase58()]: 'proofsOnlyZkApp',
+    [secondaryAddr.toBase58()]: 'secondaryZkApp',
+    [deployerAccount.toPublicKey().toBase58()]: 'deployer',
+  };
+  ```
 
-Then deploy your smart contracts as follows and visualize the transaction:
+1. Deploy your smart contracts and visualize the transaction:
```

Comment:
The code block below actually deploys the smart contract. So no user interaction is required. Maybe it could say:
```suggestion
1. Create and send a deploy transaction, then visualize it:
```
I'm not sure that's better though, I'm having trouble with words today 😂